app/javascript/components/Payouts/index.tsx

```diff
@@ -839,25 +840,18 @@ const Payouts = ({
                         )}
                       </div>
                     </div>
-                    <div>
-                      <p>Amount</p>
-                      <div>${formatPriceCentsWithoutCurrencySymbol("usd", instantPayoutAmountCents)}</div>
-                    </div>
-                    <div>
-                      <p>Instant payout fee ({INSTANT_PAYOUT_FEE_PERCENTAGE * 100}%)</p>
-                      <div>
-                        -$
-                        {formatPriceCentsWithoutCurrencySymbol("usd", instantPayoutFee)}
-                      </div>
-                    </div>
+                    <CartPriceItem title="Amount" price={instantPayoutAmountCents} />
```

Comment:
It's really odd to me that we use "cart" styling here. Maybe better to just inline the styles or create a local component, as I guess these shouldn't change in lockstep